index.bs

```diff
@@ -1158,6 +1158,10 @@ Its value is «[ [=source type/navigation=] → 3, [=source type/event=] → 1 ]
 [=aggregatable report/required aggregatable budget=] of all [=aggregatable reports=] created for
 an [=attribution source=]. Its value is 65536.
 
+<dfn>Max aggregation keys per source registration</dfn> is a positive integer that
```

Comment:
Please also remove `the user agent's` from https://wicg.github.io/attribution-reporting-api/#ref-for-max-aggregation-keys-per-source-registration.